GameServer/Logic/GameObjects/Pathfinder.cs

```diff
@@ -14,31 +14,40 @@ class Pathfinder
 
         protected static Map chart;
         private const int GRID_SIZE = 1024;
-        protected static int successes = 0, oot = 0, empties = 0;
+        protected static int successes, oot, empties;
         protected static int totalDuration = 0, durations = 0;
```

Comment:
Feel free to change this line the same way as the one above.
I'd also like to move the definitions to separate lines for the sake of consistency across the codebase.